### ISSUE TITLE
Computed property `set` is being formatted to `self.set`

### ISSUE THREAD

**Issue by qvacua** [nicklockwood/SwiftFormat#1599]
When I `swiftformat` the file `MainWindow.swift`(https://github.com/qvacua/vimr/blob/1b9545df2bbcace8f3bed4dc8907c14c3aa8f182/VimR/VimR/MainWindow.swift), the [`set` of the computed property `activateAsciiImInInsertMode`](https://github.com/qvacua/vimr/blob/1b9545df2bbcace8f3bed4dc8907c14c3aa8f182/VimR/VimR/MainWindow.swift#L37) gets formatted to `self.set`, i.e., the following

```swift
var activateAsciiImInInsertMode: Bool {
  get { self.neoVimView.activateAsciiImInNormalMode }
  set { self.neoVimView.activateAsciiImInNormalMode = newValue }
}
```

gets formatted to 

```swift
var activateAsciiImInInsertMode: Bool {
  get { self.neoVimView.activateAsciiImInNormalMode }
  self.set { self.neoVimView.activateAsciiImInNormalMode = newValue }
}
```

using the config <https://github.com/qvacua/vimr/blob/master/.swiftformat>. I'm using the version `0.52.11` installed via Brew and Swift 5.9.

I'm afraid I could not work out a minimal reproducible example.

**Comment by muradtries** [comment]
I'm having exact the same issue as you described

**Comment by nicklockwood** [comment]
@qvacua @muradtries fixed in 0.53.0
